collect_app/src/main/java/org/odk/collect/android/views/SuppressFlingGestureSlider.java

```diff
@@ -0,0 +1,42 @@
+package org.odk.collect.android.views;
+
+import android.content.Context;
+import android.util.AttributeSet;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.google.android.material.slider.Slider;
+
+import org.jetbrains.annotations.NotNull;
+
+public class SuppressFlingGestureSlider extends Slider implements  Slider.OnSliderTouchListener {
+
+    private boolean suppressFlingGesture;
+
+    public SuppressFlingGestureSlider(@NonNull Context context) {
+        super(context);
+    }
+
+    public SuppressFlingGestureSlider(@NonNull Context context, @Nullable AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    private void setSuppressFlingGesture(boolean suppressFlingGesture) {
```

Comment:
I'd maybe avoid a private setter and just go for setting the value directly. If you like it it's not a big deal to leave it in though!